CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Plutus Changelog 
+
+Changelogs are available for these packages:
+
+- [plutus-ledger-api](./plutus-ledger-api/CHANGELOG.md)
+- [plutus-tx](./plutus-tx/CHANGELOG.md)
+- [plutus-tx-plugin](./plutus-tx-plugin/CHANGELOG.md)
+- [plutus-core](./plutus-core/CHANGELOG.md)
```

Comment:
Unrelated to this PR, I find it strange that we also release `prettyprinter-configurable` but it doesn't have a changelog. @IntersectMBO/plutus-core-maintainers any idea why this is?

Reply:
We hope to extract it into a separate library and publish it on Hackage some time, so that we don't need to keep it in the Plutus repo at all.